azurerm/internal/services/policy/virtual_machine_configuration_policy_assignments_resource_test.go

```diff
@@ -0,0 +1,166 @@
+package policy_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance/check"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/policy/parse"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+type GuestConfigurationAssignmentResource struct{}
+
+func TestAccGuestConfigurationAssignment_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_virtual_machine_configuration_policy_assignment", "test")
+	r := GuestConfigurationAssignmentResource{}
+
+	data.ResourceTest(t, r, []resource.TestStep{
+		{
+			Config: r.basic(data),
+			Check: resource.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
```

Comment:
could we add a complete and update test here?

Reply:
Initially we have that during the development, I removed a lot of attribute that does not work for terraform and now everything in `basic` is all of the attributes in this resource. Therefore we cannot make another `complete` test...